### DISCUSSION TITLE
网站与文章的浏览人数及次数显示不正确问题

### DISCUSSION THREAD

**Discussion by Taootrum** [hugo-next/hugo-theme-next#73] [Q&A] [answered]
hi next团队：
第一次接触搭建博客，搭建后，网站的浏览人数以及次数等相关信息显示不正确,如图：
![image](https://user-images.githubusercontent.com/37130412/205451241-706d0329-93be-421b-b096-175e33379cae.png)

并且首页上显示的文章的浏览和评论也不正确，一直再转圈圈。
![image](https://user-images.githubusercontent.com/37130412/205451251-74e2e6d9-8c4b-4ff7-bd08-c599947b4198.png)

麻烦帮忙指导下如何修正这些问题，感谢！！

**Comment by kaolengmian7** [comment]
这两个数据由不算子 or 51LA 提供。
https://busuanzi.ibruce.info/
https://www.51.la/

在 Hugo NexT 的 config.yaml 文件下的“站点统计功能” 提供相关的配置。
配置好后，你可以在谷歌浏览器使用 F12 打开控制台看相关数据的网络请求是否正常。

**Reply by Taootrum** [reply]
感谢您的回复，由于我不是做这方向的，调试起来有点麻烦，所以在请教下：
我两篇文章，有一篇的浏览一直再转圈圈，另一篇没有问题，请问这是什么原因？
![5OEGhrWBeZ](https://user-images.githubusercontent.com/37130412/205567837-a3d71acb-b8fe-496a-a18b-e9f282d5ab11.jpg)

**Reply by kaolengmian7** [reply]
你点击进入这篇文章会展示浏览数吗？

**Reply by Taootrum** [reply]
进入文章中是有的
![image](https://user-images.githubusercontent.com/37130412/205574409-0219112c-3d9e-4ee9-bb99-aebeb32bc748.png)

**Reply by kaolengmian7** [reply]
你的网站搭建好了吗？提供一个网址我看下

**Reply by Taootrum** [reply]
https://taoo.tech/
FYI 感谢🙇‍

**Reply by kaolengmian7** [reply]
这个数据也是由第三方数据统计：不算子提供的。可能是不算子提供的接口有问题。建议做如下改动：
1. 再发一篇测试文章，看看是不是不算子只提供了一个文章的数据而不是整个文章列表的数据。
2. 我自己的网站使用的是 waline 做统计，没遇到这个问题。

**Reply by Taootrum** [reply]
好的 感谢回复 我试试

**Reply by kaolengmian7** [reply]
你这是国外站吗？

**Reply by Taootrum** [reply]
不是的 是国内站，用的腾讯云

**Comment by elkan1788** [comment, marked as answer]
@Taootrum  
- 第1个图是因为你在本地调试，而主题默认使用的是**不蒜子**统计站点访问次数，`localhost` 是大家都在使用的，所以站点统计数字会很大；
- 第2个图是文章统计和评论统计，是使用的Waline插件实现（不蒜子只支持文章访问），所以需要配置Waline服务，不然就会一直转圈圈的；

> 推荐使用 51LA作站点统计，Waline 做文章统计和评论统计

**Reply by Taootrum** [reply]
Hi elkan1788:
更换51La做站点统计后，这些信息会一直转圈圈。
![image](https://user-images.githubusercontent.com/37130412/205604928-09031f9c-2c90-424d-8e0d-e9c725c7fb1b.png)

**Reply by elkan1788** [reply]
本地开发预览使用localhost是显示不出来的，是51LA的自己的问题，你可以设置个本地的域名，然后需要在51LA后面配置可访问域名的权限，你试试看。
![image](https://user-images.githubusercontent.com/1217096/205608893-b7cc2b47-3635-44b7-9a76-2ab8634af506.png)
![image](https://user-images.githubusercontent.com/1217096/205609013-5daa6f25-8738-48c3-b2e9-54b95b21d67b.png)

**Reply by Taootrum** [reply]
Hi @elkan1788:
感谢你的指导，目前我已有国内域名：[https://taoo.tech/](https://taoo.tech/)
按照步骤操作后，还是再转圈圈，但是发现似乎是这个文件有问题：https://v6-widget.51.la/v6/Jtoq0wwFBe8yNdwG/quote.js?theme=0&col=true&f=12&display=0,0,0,1,0,1,1,1， 请帮忙指导下如何操作！感谢
![image](https://user-images.githubusercontent.com/37130412/205618343-67e4a434-bc82-46a9-b8f1-7817b6c572d0.png)

这是我51la信息：
![image](https://user-images.githubusercontent.com/37130412/205618568-07903e41-78e8-44ca-b795-6bbcad201246.png)

**Reply by Taootrum** [reply]
Hi @elkan1788 @kaolengmian7 ： 
请帮忙看下上面这个问题，十分感谢！！

**Reply by elkan1788** [reply]
上面报的是404错误，应该是你那个数据挂件没有开启，你检查一下。
![image](https://user-images.githubusercontent.com/1217096/208340733-acc6555c-ca48-45e2-a658-081f127b7f5c.png)

> 这个是文档没有写好，看来得更新一下。

**Reply by Taootrum** [reply]
@elkan1788 感谢你的支持，已经顺利解决这个问题
